docs/build/vm/intro.md

```diff
@@ -63,8 +63,20 @@ In order to install a VM, its binary must be installed in the `AvalancheGo` plug
 Multiple VMs can be installed in this location.
 
 Each VM runs as a separate process from AvalancheGo and communicates with `AvalancheGo` using gRPC
-calls. This is functionality is enabled by `rpcchainvm`, a special VM that wraps around other VM
-implementations so that they can communicate back and forth with the AvalancheGo.
+calls. This functionality is enabled by **RPCChainVM**, a special VM which wraps around other VM
+implementations and bridges the VM and AvalancheGo, establishing a standardized communication 
+protocol between them.
+
+:::info
+
+During VM creation, handshake messages are exchanged via **RPCChainVM** between AvalancheGo and the 
+VM installation. Ensure matching **RPCChainVM** protocol versions to avoid 
+[errors](https://github.com/ava-labs/avalanchego/pull/2021), by updating your VM or using a 
```

Comment:
```suggestion
errors, by updating your VM or using a 
```
we like to keep links to GH at a minimum. Linking to PRs requires them to context switch and might fatigue the reader, resulting in them quitting a tutorial before they finish. Linking to the releases is okay in this instance.